### PR TITLE
chore: move packages/components into core

### PR DESCRIPTION
**Pull request by jorgemoya**
## What/Why?
Move `packages/components` into core `components/ui`.
- Merged `package.json` and `tailwind.config` into core.
- Updated `changeset` configuration.

## Testing
Locally.

### CODE REVIEW

**Comment by jorgemoya** [comment]
Disabled Visual Regression tests to unblock this PR.

**Comment by jorgemoya** [comment]
@chanceaclark @matthewvolk If you guys can 👀 particularly the changes I did to `create-catalyst` that'd be great.

**Comment by jorgemoya** [comment]
Splitting PR into commits/multiple PRs.

File: pnpm-lock.yaml
Comment (chanceaclark) [inline pnpm-lock.yaml:1]: If you haven't already, I would make sure to rerun `pnpm install` after you removed the old `package.json`.

**Review by chanceaclark — APPROVED** [review]

**Pull request merged**